serve.sh

```diff
@@ -1,3 +1,6 @@
 #! /usr/bin/env bash
 
 runghc -isrc/ bin/serve.hs
+
+# consider switching to
```

Comment:
This seems interesting. Can you add why this would better ? Does it load faster ?

Reply:
AFAIK; running it via cabal ensures that we read the cabal file and observe things within it. As far as I know, `runghc` bypasses cabal and just loads the source you point it to, attempting to run it.